winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Changing start row and column index while exporting
+
+Export the data to the specified row index and column index in the worksheet, by setting the `StartRowIndex` and `StartColumnIndex` properties.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.StartRowIndex = 3;
+options.StartColumnIndex = 3;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-change-row-column-index.png" alt="Changing Row and Column Index while Exporting from WinUI TreeGrid to Excel" width="100%" Height="Auto"/>
+
+## Row Height and Column Width customization  
+
+After exporting the data to Excel, you can set different row heights and column widths for the columns. You can refer to [here](http://help.syncfusion.com/file-formats/xlsio/worksheet-rows-and-columns-manipulation#adjust-row-height-and-column-width) for more information.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+workBook.Worksheets[0].SetRowHeight(2, 50);
+workBook.Worksheets[0].SetColumnWidth(2, 50);
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+## Styling cells based on CellType in Excel
+
+Customize the cell styles based on the `CellType` by using the `GridExportHandler`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.GridExportHandler = GridExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void GridExportHandler(object sender, TreeGridExcelExportStartOptions e)
+{
+    if (e.CellType == ExportCellType.HeaderCell)
+    {
+        e.Style.Color = Color.Red;
+        e.Style.Font.Color = ExcelKnownColors.White;
+        e.Handled = true;
+    }
+    else if(e.CellType == ExportCellType.RecordCell)
+    {
+        e.Style.Color = Color.LightPink;
+        e.Handled = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-cell-style-customization.png" alt="WinUI TreeGrid displays Customized Cell Style based on CellType in Exported Excel" width="100%" Height="Auto"/>
+
+## Cell customization in Excel while exporting
+
+Customize the cells by setting the `CellsExportHandler` in the `TreeGridExcelExportOptions`.

```

Comment:
Customize the cells by setting the `CellsExportHandler` in the `TreeGridExcelExportOptions.`

Reply:
Updated